packages/slate-react/src/components/editable.tsx

```diff
@@ -11,7 +11,6 @@ import {
 } from 'slate'
 import { HistoryEditor } from 'slate-history'
 import throttle from 'lodash/throttle'
-import scrollIntoView from 'scroll-into-view-if-needed'
```

Comment:
Since you're no longer using the scroll-into-view-if-needed package, maybe it should also be deleted from package.json?     
https://github.com/ianstormtaylor/slate/blob/ffb6d79b66ba89db7f0687889853011a00944015/packages/slate-react/package.json#L23

By the way, I personally think this is a valuable PR that fixes a serious bug.